Sources/Optimizely/OptimizelyClient.swift

```diff
@@ -186,9 +186,9 @@ open class OptimizelyClient: NSObject {
         }
     }
     
-    func fetchDatafileBackground(resourceTimeout: Double? = nil, completion: ((OptimizelyResult<Data>) -> Void)?=nil) {
+    func fetchDatafileBackground(resourceTimeout: Double? = nil, completion: ((OptimizelyResult<Data>) -> Void)? = nil) {
         
-        datafileHandler.downloadDatafile(sdkKey: self.sdkKey, resourceTimeoutInterval: resourceTimeout) { result in
+        let processDownloadResult = { (result: OptimizelyResult<Data?>) in
```

Comment:
why?

Reply:
Refactored for reachability support but decided to move the check to other part.